addons/pos_epson_printer_restaurant/static/src/js/multiprint.js

```diff
@@ -7,8 +7,8 @@ const Registries = require('point_of_sale.Registries');
 
 // The override of create_printer needs to happen after its declaration in
 // pos_restaurant. We need to make sure that this code is executed after the
-// multiprint file in pos_restaurant.
-require('pos_restaurant.multiprint');
+// models file in pos_restaurant.
+require('pos_restaurant.models');
```

Comment:
can you rename this file from multiprint.js to models.js

Reply:
Also done it for the `pos_restaurant_iot` module